appserver/connectors/connectors-internal-api/src/main/java/org/glassfish/connectors/config/ConnectorConnectionPool.java

```diff
@@ -85,31 +85,24 @@
  *
  * There can be more than one pool for one connection-definition in one
  * resource-adapter
- * 
+ *
  */
-
-/* @XmlType(name = "", propOrder = {
-    "description",
-    "securityMap",
-    "property"
-}) */
-
 @Configured
 @ConnectionPoolConstraints({
-    @ConnectionPoolConstraint(value = ConnectionPoolErrorMessages.MAX_STEADY_INVALID)
+    @ConnectionPoolConstraint(value = ConnectionPoolErrorMessages.POOL_SIZE_STEADY),
```

Comment:
I split the common validation into two standalone. The relation between those two values is validated in max pool size validation.